object-store-rs/python/object_store_rs/fsspec.py

```diff
@@ -26,16 +26,12 @@
 
 import object_store_rs as obs
 
-if TYPE_CHECKING:
-    from object_store_rs.store import ObjectStore
-
 
 class AsyncFsspecStore(fsspec.asyn.AsyncFileSystem):
-    store: ObjectStore
 
     def __init__(
         self,
-        store: ObjectStore,
+        store,
```

Comment:
Why did you remove the `ObjectStore` typing here?

Reply:
Yeah it's a type-only object. Perhaps it should be changed in the future to be an actual Python object that can be imported at runtime. The `from __future__ import annotations` should allow this to not be evaluated.